StorageControl/src/V2/resources/storage_control_rest_client_config.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+return [
+    'interfaces' => [],
```

Comment:
By it being empty does this mean REST is not available?

Reply:
Yes. This is where the http annotations would go. It being empty means someone trying to run this with the rest transport would get a PHP fatal error.

Reply:
This is something we should be able to fix without too much trouble (see https://github.com/googleapis/gapic-generator-php/issues/705)

Reply:
I do not see a major issue into having an error thrown in case REST is specified. The library user does get an actionable insight in that case, so I think we could merge this PR and iterate with gapic-generators later.